jenkinsapi/jenkins.py

```diff
@@ -40,12 +40,13 @@ class Jenkins(JenkinsBase):
     Represents a jenkins environment.
     """
 
+    # pylint: disable=too-many-arguments
     def __init__(
             self, baseurl,
             username=None, password=None,
             requester=None, lazy=False,
             ssl_verify=True, cert=None,
-            timeout=10, useCrumb=False):
+            max_retries=None, timeout=10, useCrumb=False):
```

Comment:
If somebody uses all params in positional manner, this change will break their code. I'd rather put this new parameter as last one.

Reply:
Sure !